.husky/pre-commit

```diff
@@ -2,3 +2,15 @@
 . "$(dirname "$0")/_/husky.sh"
 
 npx lint-staged
+
+declare -a THEMES
+DATE=$(date)
+THEMES=($(git diff --cached --name-only | grep -i "theme" || :))
+
+if [ ${#THEMES[@]} -ne 0 ]; then
+  echo "📄 Date-stamping modified token theme files."
+  for i in "${THEMES[@]}"; do
+    echo "Last modified on $DATE" | cat - $i > /tmp/out && mv /tmp/out $i
```

Comment:
It writes the file correctly, but it doesn't make this a comment, so it's not a valid CSS file

Reply:
![image](https://user-images.githubusercontent.com/783820/212172175-fd616f6e-8c69-4730-938c-96f24aec8932.png)